src/NzbDrone.Core/Languages/RealLanguageFieldConverter.cs

```diff
@@ -0,0 +1,17 @@
+using System.Collections.Generic;
+using System.Linq;
+using NzbDrone.Core.Annotations;
+
+namespace NzbDrone.Core.Languages
+{
+    public class RealLanguageFieldConverter : ISelectOptionsConverter
+    {
+        public List<SelectOption> GetSelectOptions()
+        {
+            return Language.All
+                .Where(l => l != Language.Unknown && l != Language.Any)
```

Comment:
Need to exclude `Original` or handle it special in XBMCMetadata.cs

Reply:
Resolved in XBMCMetadata.